pandapower/converter/powermodels/to_pm.py

```diff
@@ -338,6 +338,8 @@ def ppc_to_pm(net, ppci):
         pm["branch"][str(idx)] = branch
 
     for idx, row in enumerate(ppci["gen"], start=1):
```

Comment:
iterating through ppci["gen"] is very slow for grids with many gens. can you vectorize?